include/dsVideoPort.h

```diff
@@ -660,6 +660,58 @@ dsError_t dsSetForceDisable4KSupport(intptr_t handle, bool disable);
  */
 dsError_t dsGetForceDisable4KSupport(intptr_t handle, bool *disable);
 
+/**
+ * @brief Enables/Disables ALLM mode for HDMI output video port.
```

Comment:
how do video port know ALLM mode is supported?
Need test cases of these API's

Reply:
Its SOC feature to enable low latency for the hdmi out port. will check if any capability API needed

Reply:
Its SOC feature to enable low latency for the hdmi out port. will check if any capability API needed